libnet/src/libnet_link_bpf.c

```diff
@@ -61,7 +63,7 @@ libnet_bpf_open(char *err_buf)
      */
     for (i = 0;;i++)
     {
-        sprintf(device, "/dev/bpf%d", i);
+        snprintf(device, 11, "/dev/bpf%d", i);
```

Comment:
`11` should be `sizeof(device)`

Reply:
also, its a pre-existing bug, but perhaps you should assert if i > 999, since the code won't work.